### PR TITLE
180 My achievements

### PR DESCRIPTION
**Pull request by namkha87**
https://trello.com/c/rTS9G6Zi/180-180-reward-achievement-system

related PR: https://github.com/anyTV/heartbeat-comments-extension/pull/188

@ravenjohn please help to run this script on accounts.freedom.tm PROD database to add callback url for heartbeat.tm 

UPDATE accounts_freedom_tm.services 
SET redirect_uri = concat(redirect_uri, ',http://heartbeat.tm/login_callback.html')
WHERE service_id = 'heartbeat';

also run when deploy: bower install

### CODE REVIEW

File: achievements/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.96.0/css/materialize.min.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <div id='loading' style='display:none'>
+            <div class='progress'>
+              <div class='indeterminate'></div>
```
Comment (nhvu1988) [inline achievements/index.html]: change to double quote

File: achievements/jsx/user_achivements.js [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+				if (current_page === this.state.current_page) {
+					return;
+				}
+
+				this.setState({
+					current_page: current_page
+				});
+
+				this.load_user_rewarded_actions(current_page);
+			},
```
Comment (nhvu1988) [inline achievements/jsx/user_achivements.js]: should refactor 3 functions on_prev_page, on_next_page and on_goto_page? they have same lines

```javascript
if (current_page === this.state.current_page) {
    return;
}

this.setState({
    current_page: current_page
});

this.load_user_rewarded_actions(current_page);
```

File: achievements/index.html (hunk 1 above)
Comment (namkha87) [inline achievements/index.html]: fixed :)

**Comment by nhvu1988** [comment]
@ravenjohn check, can merge after anyTV/heartbeat-comments-extension/pull/188 is merged

**Comment by nhvu1988** [comment]
@ravenjohn please merge this :)

**Pull request merged**